test/unit/model/jdk-install-test.js

```diff
@@ -162,7 +162,26 @@ describe('JDK installer', function() {
       });
     });
 
+    it('should create deafult empty callback if not provided', function() {
+      let jdk = new JdkInstall(installerDataSvc, 'url', 'file');
+      mockDetectedJvm('1.8.0_1');
+      try {
+        jdk.detectExistingInstall();
+      } catch (exception)  {
+        expect.fail('Did not created default empty callback');
+      }
+    });
 
+    it('should check for available msi installtion', function(done) {
+      mockDetectedJvm('1.8.0_1');
+      let jdk = new JdkInstall(installerDataSvc, 'url', 'file');
+      jdk.findMsiInstalledJava.restore();
+      return jdk.detectExistingInstall(function() {
+        expect(executeFileStub).to.have.been.calledOnce;
+        expect(writeFileStub).to.have.been.calledOnce;
```

Comment:
Thanks @jrichter1, I was thinking about ether skip it for other platforms or override in some way process.platform and let run on stabbed functions.